pkgs/applications/networking/mumble/default.nix

```diff
@@ -22,7 +22,7 @@ let
     nativeBuildInputs = [ pkgconfig python ]
       ++ { qt4 = [ qmake4Hook ]; qt5 = [ qt5.qmake ]; }."qt${toString source.qtVersion}"
       ++ (overrides.nativeBuildInputs or [ ]);
-    buildInputs = [ boost protobuf avahi ]
+    buildInputs = [ boost protobuf3_6 avahi ]
```

Comment:
Can you add a comment with a link to the issue, and that it can probably be dropped in the next release?

Reply:
is this good ?

Reply:
Perfect. It's helpful for the next person looking, and their like "why's protofuf3_6 here`.